modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+= Configure a Customer-Managed VPC on GCP
+:description: Connect Redpanda Cloud to your existing VPC resources for additional security.
+:page-cloud: true
+
+You can connect Redpanda Cloud to your customer-managed virtual private cloud (VPC), and you can then create clusters with the most security. With standard BYOC clusters using a Redpanda-managed VPC, Redpanda creates a VPC in your Google Cloud account and manages its resources. With a customer-managed VPC:
+
+* You provide your own VPC in your Google Cloud account.
+* You maintain more control of your Google Cloud account, because Redpanda requires less permissions than standard BYOC clusters.
+* You control your security resources and policies, including subnets, service accounts, IAM roles, firewall rules, and storage buckets.
+
+[NOTE]
+====
```

Comment:
Darn--to unlock they have to fill out a form? No direct email/contact? Not the best/quickest/easiest user experience.

Reply:
@bpraseed is there a better link to direct users?